sort-selection/extension.js

```diff
@@ -21,9 +21,29 @@ function activate(context) {
 }
 exports.activate = activate;
 
-function sort(txt) {
+// These codes are taken from https://stackoverflow.com/a/15479354, done by Georg Barikin (https://github.com/gebrkn)
+
+function naturalCompare(a, b) 
+{
+    var ax = [], bx = [];
```

Comment:
better to never use `var` but rather `const` or `let`

Reply:
You are right. I had to keep myself updated with ES6. Done.